content/recipes/introduction/FAIR-and-knowledge-graphs.md

```diff
@@ -180,40 +178,29 @@ for computational work.
 ##### ii. Persisting RDF graphs: RDF triple Stores
 
 RDF graph objects can be persisted in specialized databases, `RDF graph databases` also known as `RDF triple stores`. 
-To view some of the most performant and successful solutions, see the table below. There is extensive literature
-comparing the merits of the various solutions.
-
-```{dropdown} **Well known RDF triple store solutions**
-:open:
-| name | vendor | site | performance|
```

Comment:
I like the table, but there was not much content within. Also the first column was not rendered properly.